src/test/components/Timeline.test.js

```diff
@@ -488,6 +488,8 @@ describe('Timeline', function () {
   });
 
   describe('TimelineSettingsHiddenTracks', () => {
+    autoMockDomRect();
```

Comment:
You can leave it (maybe we should even put that in setup?), but I wonder why it was necessary because I see no change that seems to require it. Do you know?

Reply:
It looks like it's happening because of the timeline selection and ruler component. I think this started to happen because now we are in the same container as this component now